build.gradle

```diff
@@ -48,9 +48,10 @@ distributions {
     }
 }
 
-signing {
-    sign configurations.archives
-}
+//TODO Uncomment this before any real release
```

Comment:
How's about `Turn the signing action back on when we'll merge this into the `master`

Reply:
Much better